### ISSUE TITLE
Incorrect switch refactoring if test variable is changed in consecutive if-statements

### ISSUE THREAD

**Issue by sjvrijn** [sourcery-ai/sourcery#108]
### Issue description or question

For the following code block for example:
```python
if action == 'F':
    action = direction

if action == 'N':
    northsouth += value
elif action == 'S':
    northsouth -= value
elif action == 'E':
    eastwest += value
elif action == 'W':
    eastwest -= value
```
The suggested refactor is:
```python
if action == 'E':
    eastwest += value
elif action == 'F':
    action = direction
elif action == 'N':
    northsouth += value
elif action == 'S':
    northsouth -= value
elif action == 'W':
    eastwest -= value
```
Which is incorrect, as the `if action == 'F'` case has to be executed first, which would change the outcome.

### Sourcery Version

0.7.12

### Code editor or IDE name and version

Pycharm Professional v2020.1.1?

### OS name and version

Windows

**Comment by Hellebore** [comment]
Have reproduced - will look into addressing as soon as possible.

**Comment by Hellebore** [comment]
Fixed in next release.

**Comment by Hellebore** [comment]
Resolved with release of 0.8.6.
